src/lib/hooks/useConnect/setupConnection.ts

```diff
@@ -86,17 +91,30 @@ export async function setUpConnection({
         logger?.info?.('SendbirdProvider | useConnect/setupConnection/configureSession', sessionHandler);
         newSdk.setSessionHandler(sessionHandler);
       }
-      // to check if code is released version from rollup and *not from storybook*
-      // see rollup config file
-      // @ts-ignore
-      if (IS_ROLLUP === IS_ROLLUP_REPLACE) {
-        logger?.info?.('SendbirdProvider | useConnect/setupConnection/setVersion', APP_VERSION_STRING);
-        newSdk.addExtension('sb_uikit', APP_VERSION_STRING);
-      }
+
+      logger?.info?.('SendbirdProvider | useConnect/setupConnection/setVersion', { version: app_version });
+      newSdk.addExtension('sb_uikit', app_version);
```

Comment:
also, APP_VERSION_STRING, could have been used here, because we dont use is, better to remove from rollup.config also

Reply:
Here it is https://github.com/sendbird/sendbird-uikit-react/pull/551/commits/4568885ea6f59553358bafc46331432789494d93